compiler/rustc_target/src/spec/wasm32_wasi_preview1_threads.rs

```diff
@@ -0,0 +1,134 @@
+//! The `wasm32-wasi-preview1-threads` target is a new and still (as of July 2023) an
```

Comment:
Ie, much of this commentary should be replicated in the target docs

Reply:
✅